src/apps/bridge/rbrPressureProcessor.cpp

```diff
@@ -178,13 +178,21 @@ static void runTask(void *param) {
     }
     if (bits & kQRcv) {
       if (xQueueReceive(_ctx.q, &rbr_data, 0) == pdTRUE) {
-        if (diffSignal.addSample(rbr_data.pressure_deci_bar)) {
-          bridgeLogPrint(BRIDGE_SYS, BM_COMMON_LOG_LEVEL_INFO, USE_HEADER,
-                         "%s Added sample %.2f to Diff signal \n", kRbrPressureProcessorTag,
-                         rbr_data.pressure_deci_bar);
+        if (_ctx.started) {
+          if (diffSignal.addSample(rbr_data.pressure_deci_bar)) {
```

Comment:
This is the most important change — checking `_ctx.started` to make sure the timer is currently running before adding a reading to the difference signal buffer.